.github/workflows/build_publish_lambda_layer.yml

```
@@ -66,16 +75,38 @@ jobs:
       fail-fast: false
```

Unless there's a particular reason I think we should have this job fail fast.

---

Updated.